src/main/java/uk/gov/hmcts/cmc/claimstore/services/OffersService.java

```diff
@@ -50,7 +53,9 @@ public void accept(Claim claim, MadeBy party) {
         settlement.accept(party);
 
         offersRepository.acceptOffer(claim.getId(), jsonMapper.toJson(settlement), LocalDateTime.now());
-        eventProducer.createOfferAcceptedEvent(claim, party);
+        Claim claimById = claimService.getClaimById(claim.getId());
+        System.out.println(claimById);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS106)